Dockerfile

```diff
@@ -38,8 +38,10 @@ ARG opa_url
 ARG candig_auth
 ARG minio_url
 ARG minio_bucket_name
+ARG vault_url
 RUN sed -i s@\<CANDIG_OPA_SECRET\>@${opa_secret}@ config.ini \
```

Comment:
Adding secrets in at build is not recommended. If this image is published to repo, the secret is saved inside the image.

Reply:
I fixed this in a later commit.